analyses/copy_number_consensus_call/run_consensus_call.sh

```diff
@@ -16,11 +16,11 @@ SCRATCHDIR=../../scratch/copy_consensus
 mkdir -p $SCRATCHDIR
 mkdir -p results
 
-## Run the python script to go from 1 big manta file, cnvkit file and freec file into 3 directories. 
+## Run the python script to go from 1 big gatk file, cnvkit file and freec file into 3 directories.
 ## Each directory with individual sample files.
 
 python3 scripts/merged_to_individual_files.py \
-    --manta ../../data/sv-manta.tsv.gz \
+    --gatk ../../data/merged_gatk_cnv_called_copy_ratio_segments_tumor.seg \
```

Comment:
```suggestion
    --gatk ../../data/cnv-gatk.seg.gz \
```
let's use this as the filename to be consistent with the other cnv files. @migbro @dmiller15 and/or @zhangb1 - can you confirm the format used will be our standard merged format (I think this file is only local to you @sickler-alex)? @sickler-alex can you point us to this file on cavatica?

Reply:
https://cavatica.sbgenomics.com/u/kfdrc-harmonization/sd-8y99qzjj-ad-hoc-gatk-cnv-wxs/files/62d045d008804a1f3133d7d2/

Reply:
`cnv-gatk.seg.gz ` is good I think ,with other we use `cnv-cnvkit.seg.gz
cnv-controlfreec.tsv.gz`. for cnvkit and controlfreec.....  but for the format.. I am not sure ,, I don't have the answer ..